sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/TestCases/ImplementIDisposableCorrectly.cs

```diff
@@ -17,6 +17,17 @@ public void Dispose()
         protected virtual void Dispose(bool disposing) { }
     }
 
+    public class SimpleDisposableWithSuppressFinalize : IDisposable

```

Comment:
Should SimpleDisposable (at line 10) require call to GC.SuppressFinalize, as per links provided in the ticket?